client/src/firebase.js

```diff
@@ -0,0 +1,16 @@
+import { initializeApp } from "firebase/app";
+import { getAuth } from "firebase/auth";
+
+const firebaseConfig = {
```

Comment:
Can we get this added as a local credentials file (added to .gitignore)? anybody from the public can see these otherwise.

Example: https://github.com/logan-wood/notepad.io/blob/main/server/services/firebase.js

Reply:
certainly. my bad, was testing firebase implementation and forgot to move these

Reply:
Adding onto this, there should be login credentials already in the server folder. Update your branch from main and have a look at server/services/firebase.js and see if it can be used in frontend. Probably a good idea to make sure this branch is up to date with main before continuing with the PR regardless